theano/scalar/basic.py

```diff
@@ -161,7 +162,7 @@ class Scalar(Type):
     TODO: refactor to be named ScalarType for consistency with TensorType.
 
     """
-
+    __props__ = ('dtype',)
```

Comment:
Yes it does because it inherits from object2 (which not has the \__props__ implementation).